routingpy/routers/google.py

```diff
@@ -319,12 +322,40 @@ def directions(  # noqa: C901
         if transit_routing_preference:
             params["transit_routing_preference"] = transit_routing_preference
 
-        return self.parse_direction_json(
+        return self._parse_direction_json(
             self.client._request("/directions/json", get_params=params, dry_run=dry_run), alternatives
         )
 
-    @staticmethod
-    def parse_direction_json(response, alternatives):
+    def _time_object_to_aware_datetime(self, time_object):
+        timestamp = time_object["value"]
+        dt = datetime.datetime.fromtimestamp(timestamp)
+        timezone = pytz.timezone(time_object["time_zone"])
+        return dt.astimezone(timezone)
```

Comment:
can we make these new methods util methods instead? I feel it's better to outsource those functions (here and otp) so they can be re-used. I'll do valhalla anyways and will PR to your PR, then you can see if you like that too:)

Reply:
This one in particular is specific to Google because the input format is theirs. Do we output to a Google-specific utility file?

Reply:
I tried to generalize it a bit in https://github.com/khamaileon/routingpy/pull/1/files